app/pages/authorisedOfficials/AddedAuthorisedOfficialPage.scala

```diff
@@ -19,10 +19,9 @@ package pages.authorisedOfficials
 import pages.QuestionPage
 import play.api.libs.json.JsPath
 
-case object AddedOneAuthorisedOfficialPage extends QuestionPage[String] {
+case class AddedAuthorisedOfficialPage(index:Int) extends QuestionPage[String] {
 
-  override def path: JsPath = JsPath \ toString
+  override def path: JsPath =  AuthorisedOfficialsId(index).path \ toString
```

Comment:
my brain's not fully working atm, but just to make sure, this won't affect anything in the submission transformers, will it?